components/dropdown/dropdown-content-mixin.js

```diff
@@ -762,9 +762,11 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 			this.dispatchEvent(new CustomEvent('d2l-dropdown-position', { bubbles: true, composed: true }));
 		};
 
-		const scrollWidth = Math.max(header.scrollWidth, content.scrollWidth, footer.scrollWidth);
+		if (!scrollPositionChange || !this._scrollWidth || !this._width) {
+			this._scrollWidth = Math.max(header.scrollWidth, content.scrollWidth, footer.scrollWidth);
```

Comment:
Solution - only check the `scrollWidth` when `__position` is called not because of scrolling.

Question - I believe that we wouldn't want resizing of the dropdown to occur at all when `__position` is called due to scrolling. If this is the case then I think we could avoid running some other calculations above as well. Is this a change I should make here or should I keep this simple and do that at another point (and also perhaps flag it)?

Reply:
> Question - I believe that we wouldn't want resizing of the dropdown to occur at all when __position is called due to scrolling.

I'm uncertain of that. I think this would be a change in behaviour. Probably worth checking with Jeff about this.

We currently have the `GAUD-7472-dropdown-popover` in play (turned off). The positioning code is effectively applied both in this `DropdownContentMixin` and `DropdownPopoverMixin` (via use of `PopoverMixin`). Once we've turned the flag on and are comfortable removing it, `DropdownContentMixin` will be deleted. We'll want to apply/validate whatever fix we want in `PopoverMixin` as well. The popover case can be checked by applying `?popover=true` to the demo page querystring.